tsdb/head.go

```diff
@@ -2288,6 +2288,7 @@ type memSeries struct {
 	ref           uint64
 	lset          labels.Labels
 	mmappedChunks []*mmappedChunk
+	mmMaxTime     int64
```

Comment:
Can we add a comment here that this is only used for WAL replay and is not updated after that?